### DISCUSSION TITLE
Working of an own implementation of your books example application

### DISCUSSION THREAD

**Discussion by hindermath** [holgerflick/flixaccounting#20] [Show and tell]
I'm working on an own implementation of the example application from your book. The progress can be seen under this repo https://github.com/hindermath/thinderaccounting
For this I'm using RAD Studio and DELPHI 12 and the appropriated TMS Software components used in the book that I have licensed. So, this is a good start for me to tie together all these components into a good real-world example application.
Thank you, Holger, for authoring this book with all these modern technologies in DELPHI!!! This shows and approves that DELPHI is still up to date in this days.

**Comment by holgerflick** [comment]
Thanks for the kind feedback.
